src/Symfony/Component/Form/README.md

```diff
@@ -19,8 +19,8 @@ https://symfony.com/doc/2.3/book/forms.html
 Resources
 ---------
 
-You can run the unit tests with the following command:
-
-    $ cd path/to/Symfony/Component/Form/
-    $ composer install
-    $ phpunit
+  * [Documentation](https://symfony.com/doc/current/components/form/index.html)
```

Comment:
there is a documentation url (for the 2.3 doc, and not a link) above, which should be removed.

Mentionning the Silex integration also looks weird